### PR TITLE
EBMac v1.3.7 added

### PR DESCRIPTION
**Pull request by stonerl**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/ebmac.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'ebmac' do
+  version '1.37'
+  sha256 '120b9e1e922d32caf60424c1561b3eb76d7eb8c9e82ad9cd976a53496e514bb7'
+
+  # ftp.vector.co.jp was verified as official when first introduced to the cask
+  url 'http://ftp.vector.co.jp/67/18/938/EBMac1.37.dmg'
```
Comment (fanquake) [inline Casks/ebmac.rb]: You can use #{version} in the url stanza

**Comment by stonerl** [comment]
I know but the URL changes with every version. `67/18/938` is only valid for this version AFAIK. I so I did this on purpose

**Comment by stonerl** [comment]
So, shall I use #{version} in the url stanza or not?

**Comment by fanquake** [comment]
You can use version here `/EBMac#{version}.dmg'`

**Comment by stonerl** [comment]
Done.

**Pull request merged**